src/Sarif.Viewer.VisualStudio.ResultSources.GitHubAdvancedSecurity/GitHubInfoBarHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.Imaging;
+using Microsoft.VisualStudio.Imaging.Interop;
+
+namespace Microsoft.Sarif.Viewer.ResultSources.Domain
+{
+    internal static class GitHubInfoBarHelper
+    {
+        internal static ImageMoniker GetInfoBarImageMoniker()
+        {
+            return KnownMonikers.StatusWarning;
```

Comment:
>StatusWarning [](http://example.com/codeflow?start=33&length=13)

why the same function in Sarif.Viewer.VisualStudio.ResultSources.GitHubAdvancedSecurity.2022 returns a different value (KnownMonikers.GitHub)?